scripts/feature_tests/upgrade/common.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
```

Comment:
Depending which PR will go in first, if it is the case for this PR, you have to update this file based on comments from [here](https://github.com/metal3-io/metal3-dev-env/pull/357/files?file-filters%5B%5D=.sh#diff-d1b9e87696eb96d44a3ae16774dd009aR1)

Reply:
This file is also duplicated with https://github.com/metal3-io/metal3-dev-env/pull/357/files#diff-043c763a6538bc09c662d10d94a69e55R3-R8

Reply:
this is a known issue and will be handled during the merge towards master branch.